pkgs/development/libraries/libbfd/default.nix

```diff
@@ -32,7 +32,7 @@ stdenv.mkDerivation {
 
   strictDeps = true;
   nativeBuildInputs = [ autoreconfHook bison ];
-  buildInputs = [ libiberty zlib.dev ];
+  buildInputs = [ libiberty zlib ] ++ lib.optionals stdenv.isDarwin [ gettext ];
```

Comment:
```suggestion
  buildInputs = [ libiberty zlib ] ++ lib.optionals stdenv.isDarwin [ libintl ];
```
makes the intent clearer IMO

Reply:
As I suggested above, with `libintl` you could even drop the conditional... but the end result will be conditional `gettext` anyway, so I personally don't care much.

Reply:
I kept the conditional because libintl doesn't seem to be needed on musl for some reason.

Reply:
Oh, in that case... shouldn't we amend the `libintl` definition?  (regardless of this particular case)

Reply:
Yeah I couldn't say, I only tried it with libbfd.

Reply:
Ok looks like you're right, we should update the libintl definition. The O.G. libintlOrEmpty did this correctly:
https://github.com/NixOS/nixpkgs/blob/8a9595cfc03e00c1b4883cb001c9ee63a18edc1e/pkgs/top-level/aliases.nix#L424
https://github.com/NixOS/nixpkgs/pull/37012#issuecomment-373187336

Reply:
ok, switched to libintl